var/spack/repos/spack_repo/builtin/packages/py_pyogrio/package.py

```diff
@@ -16,15 +16,21 @@ class PyPyogrio(PythonPackage):
 
```

Comment:
same here, can add me as maintainer if you like

Reply:
you already are a maintainer on this one